.github/workflows/shared-build-wasm.yml

```
@@ -0,0 +1,96 @@
+name: Shared / Build Wasm
+
+# This workflow builds and optionally unit tests the wasm bindings.
+# It outputs build artifacts used for publishing and building in the 'shared-build-and-test-stronghold-nodejs' workflow and for testing in 'build-and-test' workflow.
+
+on:
+  workflow_call:
+    inputs:
+      run-unit-tests:
+        description: "Should tests be run"
+        required: false
+        type: boolean
+        default: true
+      output-artifact-name:
+        description: "Name used for the output build artifact"
+        required: true
+        type: string
+jobs:  
+  build-wasm:
+    defaults:
+      run:
+        working-directory: bindings/wasm
+        shell: bash
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest ]
+        include:
+          - os: ubuntu-latest
+            sccache-path: /home/runner/.cache/sccache
+    env:
+      SCCACHE_DIR: ${{ matrix.sccache-path }}
+      RUSTC_WRAPPER: sccache
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: './.github/actions/rust/rust-setup'
+        name: rust-setup
```

Nit: name should be consistent with other occurrences.

```suggestion
      - name: Setup Rust and cache
        uses: './.github/actions/rust/rust-setup'
```